### DISCUSSION TITLE
chmod: cannot access '/opt/Insomnia/chrome-sandbox': No such file or directory

### DISCUSSION THREAD

**Discussion by joshuataylor** [Kong/insomnia#3950] [Help] [answered]
**Describe the bug**
Trying to install `Insomnia.Designer-2020.1.3.deb` on Ubuntu 20.04 shows the following:

```
Selecting previously unselected package insomnia-designer.
(Reading database ... 616674 files and directories currently installed.)
Preparing to unpack .../Insomnia.Designer-2020.1.3.deb ...
Unpacking insomnia-designer (2020.1.2) ...
Setting up insomnia-designer (2020.1.2) ...
chmod: cannot access '/opt/Insomnia Designer/chrome-sandbox': No such file or directory
Error in file "/usr/share/applications/org.kde.kdeconnect_open.desktop": "*/*" is an invalid MIME type ("*" is an unregistered media type)
Processing triggers for hicolor-icon-theme (0.17-2) ...
Processing triggers for gnome-menus (3.36.0-1ubuntu1) ...
Processing triggers for desktop-file-utils (0.24-1ubuntu2) ...
Processing triggers for mime-support (3.64ubuntu1) ...
```
**To Reproduce**
1. Run Ubuntu 20.04
2. `dpkg -i Insomnia.Designer-2020.1.3.deb`
3. See error

**Expected behavior**
It installs?

**Desktop (please complete the following information):**
 - OS: Ubuntu 20.04
 - Installation Method: `dpkg -i`
 - App Version: 2020.1.3

**Comment by sonicyeti** [comment]
Thanks for the catch and level of detail provided, we're on it! ❤️

**Comment by burtonr** [comment]
Not sure if this helps, but I was having trouble running the Insomnia REST client (not designer) with a similar error on Ubuntu freshly updated to 20.04:
```sh
$ insomnia
/snap/insomnia/62/insomnia: error while loading shared libraries: libgtk-3.so.0: cannot open shared object file: No such file or directory
```
As you can see, this was installed via snap. It was working fine, but once upgrading to Ubuntu 20.04, the app no longer started.

To "fix" it, I removed the snap and re-installed:

```sh
$ sudo snap remove insomnia 
insomnia removed
$ sudo snap install insomnia
insomnia 7.1.1 from Gregory Schier (gschier1990) installed
$
$ insomnia 
Gtk-Message: 19:15:50.024: Failed to load module "appmenu-gtk-module"
Error org.freedesktop.DBus.Error.Failed: cannot use setting "default-url-scheme-handler": not allowed
[fix] Running database repairs
[db] Initialized DB at /home/burtonr/snap/insomnia/62/.config/Insomnia/insomnia.$TYPE.db
Init responses DB
[localstorage] Initialized at /home/burtonr/snap/insomnia/62/.config/Insomnia/localStorage
[main] Loading file:///snap/insomnia/62/resources/app.asar/renderer.html
[updater] Updater not running platform=linux dev=false
```
Working fine after that (lost my data though :frowning_face: )

**Comment by sonicyeti** [comment]
We are exploring another installation delivery approach for linux but haven't an ETA on when that may ship. Thanks for the additional notes @burtonr!

**Comment by tavareshenrique** [comment]
Same problem here

**Comment by stevenossa21** [comment]
Same problem. Im waiting for new replies

**Comment by arjunbazinga** [comment]
same, bumping issue

**Comment by tavareshenrique** [comment]
I am upgrading my Insomnia to `Version 2020.3.3` and now it looks stable.

**Comment by armandoSandino** [comment]
I'm getting the same problem in version 2020.3.3 on Ubuntu 16

**Comment by MartinX3** [comment]
Fresh installed 2020.3.3 via PPA on a fresh installed Ubuntu 20.04
Getting the same error

**Comment by zigiiprens** [comment]
Installation on Ubuntu 20.04.1 get the same `chmod: cannot access '/opt/Insomnia/chrome-sandbox': No such file or directory` error from PPA source. 
`version 2020.3.3`

**Comment by jmoniatte** [comment, marked as answer]
I was having a similar issue on Debian and was able to fix it with:

```
sudo chmod 4755 /opt/Insomnia/chrome-sandbox
```

(If you try to launch insomnia with `./insomnia`, the console throws an error with instructions for fixing it)

**Reply by graphpudding** [reply]
how you  can change chmod if you havent got a folder???
